label_studio/projects/api.py

```diff
@@ -920,3 +922,38 @@ def delete(self, request, *args, **kwargs):
         count = project.delete_predictions(model_version=model_version)
 
         return Response(data=count)
+
+
+@method_decorator(
+    name='get',
+    decorator=extend_schema(
+        tags=['Projects'],
+        summary='List annotators for project',
+        description='Return users who have submitted annotations in the specified project.',
+        responses={
+            200: OpenApiResponse(
+                description='List of annotator users',
+                response=UserSimpleSerializer(many=True),
+            )
+        },
+        extensions={
+            'x-fern-sdk-group-name': 'projects',
+            'x-fern-sdk-method-name': 'annotators',
+            'x-fern-audiences': ['public'],
+        },
+    ),
+)
```

Comment:
please add a quick test for this once Spectacular / fern finish their work

Reply:
yep I will